languages/en/_pygments/tuleap_style.py

```diff
@@ -0,0 +1,19 @@
+def setup(app):
+    return
+
+from pygments.style import Style
+from pygments.token import *
+
+class TuleapStyle(Style):
+    styles = {
+        Token: '#282621',
+
+        Keyword: '#0480bf',
+        Name: '#7c2db3',
+        Name.Constant: '#f18e06',
+        String: '#6abf1d',
+        Number: '#ff8a8a',
+        Literal: '#5d5d5d',
+        
+        Comment: '#717171',
```

Comment:
* Use -text color instead of the primary version
* Add name of color in comment to know where it comes from
* Add class name of the token in comment to ease debugging
* Add `Name.Other` to have better contrast in javascript code (examples in REST API)
* Add `Comment.Preproc` to have same syntax color for CDATA than in Tuleap (project import format)
* Add `Punctuation` to have same syntax color than in Tuleap

```suggestion
        Token: '#282621',           # typo-default-text-color

        Keyword: '#005f90',         # class:'k'  deep-blue-text
        Name: '#6a14a7',            # class:'n'  plum-crazy-text
        Name.Constant: '#945600',   # class:'no' clockwork-orange-text
        Name.Other: '#774a0a',      # class:'nx' teddy-brown-text
        String: '#137900',          # class:'s'  neon-green-text
        Number: '#bf4747',          # class:'m'  coral-pink-text
        Literal: '#525252',         # class:'l'  inca-silver-text

        Comment: '#717171',         # class:'c'  dimmed-color
        Comment.Preproc: '#1b805e', # class:'cp' surf-green-text
        Punctuation: '#717171',     # class:'p'  dimmed-color

```